api/tests/unit/infrastructure/exports/campaigns/campaign-profile-collection-result-line_test.js

```diff
@@ -0,0 +1,588 @@
+              sharedAt: new Date('2019-03-01T23:04:05Z'),
+              participantExternalId: '-Mon mail pro',
+              userId: 123,
+              participantFirstName: 'Juan',
+              participantLastName: 'Carlitos',
+              division:'3eme',
+            };
+  
+            const csvExcpectedLine = `"${organization.name}";` +
+          `${campaign.id};` +
+          `"${campaign.name}";` +
+          `"${campaignParticipationResultData.participantLastName}";` +
+          `"${campaignParticipationResultData.participantFirstName}";` +
+          '"Non";' +
+          '"NA";' +
+          '"NA";' +
+          '"NA";' +
+          '"NA";' +
+          '"NA";' +
+          '"NA";' +
+          '"NA";' +
+          '"NA"' +
+          '\n';
+  
+            //when 
+            const line = new CampaignProfileCollectionResultLine(campaign, organization, campaignParticipationResultData, competences, placementProfile);
+  
+            //then
+            expect(line.toCsvLine()).to.equal(csvExcpectedLine);
+          });
+        });
+      });
+    });
+
+    context('When organization is SUP and managing students', () => {
+      beforeEach(() => {
+        organization.isSup = true;
+        organization.isManagingStudents = true;
+      });
+
+      context('when the participant does not have a student number', () => {
+        it('should return the csv without student number information', () => {
+          //given
+          sinon.stub(PlacementProfile.prototype, 'isCertifiable').returns(false);
+          sinon.stub(PlacementProfile.prototype, 'getCertifiableCompetencesCount').returns(0);
+
+          campaign.id = 123;
+          campaign.name = 'test';
+          organization.name = 'Umbrella';
+
+          const campaignParticipationResultData = {
+            id: 1,
+            isShared: true,
+            isCompleted: true,
+            createdAt: new Date('2019-02-25T10:00:00Z'),
+            sharedAt: new Date('2019-03-01T23:04:05Z'),
+            userId: 123,
+            participantFirstName: 'Juan',
+            participantLastName: 'Carlitos',
+            studentNumber: null,
+          };
+
+          const csvExcpectedLine = `"${organization.name}";` +
+        `${campaign.id};` +
+        `"${campaign.name}";` +
+        `"${campaignParticipationResultData.participantLastName}";` +
+        `"${campaignParticipationResultData.participantFirstName}";` +
+        '"";' +
+        '"Oui";' +
+        '2019-03-01;' +
+        '13;' +
+        '"Non";' +
+        '0;' +
+        '1;' +
+        '9;' +
+        '0;' +
+        '4' +
+        '\n';
+
+          //when 
+          const line = new CampaignProfileCollectionResultLine(campaign, organization, campaignParticipationResultData, competences, placementProfile);
+
+          //then
+          expect(line.toCsvLine()).to.equal(csvExcpectedLine);
+        });
+
+        it('should return the csv with student number information', () => {
```

Comment:
il manque un contexte (et il faut ferme le contexte précédent ici : 
```suggestion
    context('when the participant has a student number', () => {
        it('should return the csv with student number information', () => {
```